examples/react-native-expo/App.js

```diff
@@ -4,6 +4,8 @@ import AsyncStorage from '@react-native-async-storage/async-storage'
 import Uppy from '@uppy/core'
 import Tus from '@uppy/tus'
 import FilePicker  from '@uppy/react-native'
+// todo broken?
+// eslint-disable-next-line import/no-unresolved
 import  useUppy  from '@uppy/react/lib/useUppy'
```

Comment:
Seems like I missed this but `useUppy` does not exist so this hook should be removed from the example.

Reply:
#5405